R/mainTests.r

```diff
@@ -1152,6 +1160,483 @@ country_code <- function(data) {
   return(output)
 }
 
+
+# school_laestab_duplicates  ----------------------------------------
+# check that there is a 1:1 relationship between school laestab codes and names
+school_laestab_duplicates <- function(data) {
+  if (!"School" %in% unique(data$geographic_level)) {
+    output <- list(
+      "message" = "School-level data is not present in this data file.",
+      "result" = "IGNORE"
+    )
+  } else if ("School" %in% unique(data$geographic_level) & !"school_laestab" %in% names(data)) {
+    output <- list(
+      "message" = "School LAESTAB data must be present when including school-level data.",
+      "result" = "FAIL"
+    )
+  } else if ("School" %in% unique(data$geographic_level) & !"school_name" %in% names(data)) {
+    output <- list(
+      "message" = "School name data must be present when including school-level data.",
+      "result" = "FAIL"
+    )
+  } else if ("School" %in% unique(data$geographic_level) & !"school_name" %in% names(data) & !"school_laestab" %in% names(data)) {
+    output <- list(
+      "message" = "School LAESTAB data must be present when including school-level data.",
```

Comment:
should this be a different message?

Reply:
"school_laestab and school_name are missing from the data file. These columns are required for school level data." ? @lauraselby

Reply:
I think so!